jsettlers.main.android/src/main/java/jsettlers/main/android/gameplay/presenters/Building.java

```diff
@@ -0,0 +1,57 @@
+package jsettlers.main.android.gameplay.presenters;
+
+import jsettlers.common.buildings.EBuildingType;
+import jsettlers.common.map.partition.IBuildingCounts;
+import jsettlers.graphics.localization.Labels;
+
+/**
+ * Created by tompr on 29/05/2017.
+ */
+
+public class Building {
```

Comment:
Can you name this class so it is easily distinguished from the Building class in the game logic? Maybe something like UIBuildingInformation?